components/style/properties.mako.rs

```diff
@@ -1624,7 +1664,12 @@ pub mod longhands {
 
     // CSS 2.1, Section 15 - Fonts
 
-    ${new_style_struct("Font", is_inherited=True)}
+    ${new_style_struct("Font", is_inherited=True, gecko_name="nsStyleFont",
```

Comment:
Remember that the following Gecko structs still have incorrect layout, so I expect you to have a bad day if you actually use them:
```
FontFamilyList
StyleAnimation
nsFont
nsStyleBackground
nsStyleDisplay
nsStyleFont
nsStyleGridTemplate
nsStylePosition
nsStyleSVG
```

I have a patch in the oven though